includes/Admin/Editor.php

```diff
@@ -353,9 +353,14 @@ public function get_editor_settings() {
 	 * @return void
 	 */
 	protected function setup_lock( $story_id ) {
+		if ( ! $this->experiments->is_experiment_enabled( 'enablePostLocking' ) ) {
+			return;
+		}
```

Comment:
What is this change?

Reply:
This prevents marking a story as locked if the experiment is not enabled.

Without this, one could edit a story, and on "All Stories" it would have been shown as locked, but the locked modal was not shown in the editor.

The whole post locking should only work if the experiment is enabled.